app/tests/algorithms_tests/test_api.py

```diff
@@ -131,3 +134,27 @@ def test_result_api_permissions(client):
             content_type="application/json",
         )
         assert response.status_code == test[1]
+
+
+@pytest.mark.django_db
+def test_job_create(client):
```

Comment:
Could you add a test to check for the permissions of this view?

Reply:
Tests parameterized